GDAPI/GDAPI/Application/Editors/Editor.cs

```diff
@@ -1487,6 +1595,35 @@ private void AdjustBlocks(GeneralObject o, SourceTargetRange r)
             if (o is IHasSecondaryBlockID s && r.IsWithinSourceRange(s.SecondaryBlockID))
                 s.SecondaryBlockID += d;
         }
+
+#nullable enable
+        private LevelIDAdjustmentFunction? GetLevelIDAdjustmentFunction(LevelObjectIDType type)
+        {
+            return type switch
+            {
+                LevelObjectIDType.Color => AdjustColorChannels,
+                _ => null,
+            };
+        }
```

Comment:
Please use body expression whenever possible.

Reply:
The line just gets too long, which is why I didn't use a body expression here.